tests/unit/test_model.py

```
@@ -7,9 +8,37 @@ def test_create_account():
     WHEN a new Account is created
     THEN check the name, account_number, balance, currency, status and created_at fields are defined correctly
     """
-    account = Account('John Doe', '€')
+    account = Account('John Doe', '€', 'ESP')
     assert account.name == 'John Doe'
     assert account.currency == '€'
     assert account.account_number != None
```

Ruff

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)